spec/filters/split_spec.rb

```diff
@@ -56,4 +57,49 @@
     end
   end
 
+  describe "split array" do
+    config <<-CONFIG
+      filter {
+        split {
+          field => "array"
+        }
+      }
+    CONFIG
+
+    sample("array" => ["big", "bird", "sesame street"], "untouched" => "1\n2\n3") do
+      insist { subject.length } == 3
+      subject.each do |s|
+         insist { s["untouched"] } == "1\n2\n3"
+      end
+      insist { subject[0]["array"] } == "big"
+      insist { subject[1]["array"] } == "bird"
+      insist { subject[2]["array"] } == "sesame street"
+    end
+  end
+
+  describe "split array into new field" do
+    config <<-CONFIG
+      filter {
+        split {
+          field => "array"
+          target => "element"
+        }
+      }
+    CONFIG
+
+    sample("array" => ["big", "bird", "sesame street"]) do
+      insist { subject.length } == 3
+      insist { subject[0]["element"] } == "big"
+      insist { subject[1]["element"] } == "bird"
+      insist { subject[2]["element"] } == "sesame street"
+    end
+  end
+
+  describe "Split Filter" do
```

Comment:
this describe block is similar to its parent "describe LogStash::Filters::Split"

maybe 
```ruby
context "when invalid type is passed" do
  it "should raise exception" do
   # ...
  end
end
```